app/controllers/concerns/can_redirect_form_after_renew.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module CanRedirectFormAfterRenew
```

Comment:
I'm a little confused about when this is needed – don't we already have this in the ResourceController?